docs/en/siem/rules-ui-create.asciidoc

```diff
@@ -70,25 +71,41 @@ ones.
 [[create-rule-ui]]
 === Create a new rule
 
+You can create two types of rules:
+
+* *Custom query*: Query-based rule, which searches the defined indices and creates a signal when a document matches the rule's query.
+* *Machine learning*: {ml-cap} rule, which creates a signal when a {ml} job discovers an anomaly above the defined threshold (see <<machine-learning>>).
+
+IMPORTANT: You can only create {ml} jobs when you have a
+https://www.elastic.co/subscriptions[Platinum License] or you are using a
+{ess-trial}[Cloud] deployment.
+
 . Go to *SIEM* -> *Detections* -> *Manage signal detection rules*.
 . Click *Create new rule*.
 +
 The *Create new rule* page is displayed.
 [role="screenshot"]
-image::create-new-rule.png[]
-. Define which {es} indices the rule analyzes for signals.
-. Use the filter and query fields to create the criteria used for detecting 
+image::images/create-new-rule.png[]
+. Do *one* of the following:
+* To create a rule based on a {ml} anomaly threshold, select _Machine Learning_
+and then:
+.. The required {ml} job.
+.. The anomaly score threshold above which signals are created.
+* To create a rule base on a KQL or Lucene query, select _Custom query_ and
```

Comment:
I think you want `base` to be `based` here.

```suggestion
* To create a rule based on a KQL or Lucene query, select _Custom query_ and
```

Reply:
sure do - thanks @dhurley14